features/environment.py

```diff
@@ -331,6 +331,8 @@ def from_environ(cls, config) -> "UAClientBehaveConfig":
 def before_all(context: Context) -> None:
     """behave will invoke this before anything else happens."""
     context.config.setup_logging()
+    logging.getLogger("botocore").setLevel(logging.INFO)
+    logging.getLogger("boto3").setLevel(logging.INFO)
```

Comment:
you can remove this when resolving conflicts as I added a better version in #2260